### ISSUE TITLE
Forgetting disappears when using more examples.

### ISSUE THREAD

**Issue by suanrong** [mtoneva/example_forgetting#1]
In your continual learning setup experiment，There are only 5k examples in each task. I create two task by sampling half of the training set ( 25k examples each). Then the phenomenon in Figure4(a) disappear.

I think the forgetting is caused by severely overfitting.(In your experiement, task1 get 90% accuracy  while task2 only get 60% accuracy)

**Comment by mtoneva** [comment]
The experiments in Figure 4 are meant to contrast the learning dynamics of a random partitioning of 10k examples (in Figure 4a) with those of a partitioning of the same examples based on forgetting statistics (in Figure 4b and 4c). So in essence what we care about is not the amount of forgetting between random task 1 and random task 2, but how it compares to the forgetting between forgettable task 1 and unforgettable task 2 (and vice versa).

Also, the experiments in Fig 4b and 4c suggest that the forgettable examples are able to support the unforgettable ones (i.e. when training on forgettable examples, performance on unforgettable examples does not deteriorate as much), whereas the unforgettable examples are not able to support the forgettable ones. From this, we would expect that a random partitioning of a dataset with higher percentage of unforgettable examples would exhibit more forgetting than a dataset with a lower percentage of unforgettable examples. The 10k examples in the Fig 4 experiments contain 50% (5k) unforgettable and 50% (5k) forgettable examples, whereas the full training set is only about 30% unforgettable. So your finding that using the whole training set to create the random partitions exhibits less forgetting than the experiments in Figure 4 is in line with our findings.
